arches/app/templates/views/graph/graph-settings.htm

```diff
@@ -166,7 +166,7 @@ <h3 class="panel-title card-main-title">{% trans "Settings" %}</h3>
                                                         </div>
 
                                                         <div class="col-xs-12">
-                                                            <select class="design" {% if node_count > 1 %}disabled{% endif %} data-bind="value: ontologyClass, valueAllowUnset: false, options: ontologyClasses, optionsText: 'display', optionsValue:'source', chosen: {disable_search_threshold: 10, width: '100%', search_contains: true}"></select>
+                                                            <select class="design" {% if node_count > 1 %}disabled{% endif %} data-bind="value: ontologyClass, valueAllowUnset: false, options: ontologyClasses, optionsText: {% trans 'display' %}, optionsValue:'source', chosen: {disable_search_threshold: 10, width: '100%', search_contains: true}"></select>
```

Comment:
**optionsText** should NOT be wrapped in a trans tag, as this is a keyword in code